packages/ui/src/components/icons/types.ts

```diff
@@ -2,4 +2,5 @@ import React from 'react'
 
 export type IconProps = React.SVGAttributes<SVGElement> & {
 	size?: string | number
+	strokeWidth?: string | number
```

Comment:
Same here - think it makes sense to revert this, assuming it is not needed, and supported by all our icons?

Reply:
Per the design IconOutlineX shall have strokeWidth of 8. How else can I do the same?

Reply:
Keeping the discussion in https://github.com/highlight/highlight/pull/6682/files#r1338665209